api/application.ts

```diff
@@ -91,6 +93,10 @@ app.get('/api/configuration', (req, res) => {
     res.send(showFeature(req.query.configurationKey))
 })
 
+// TODO: move to proxy route as below
```

Comment:
can these routes be proxied? If so we can remove the routers and proxy through, otherwise remove this TODO